clients/edgex-foundry/device_client.go

```diff
@@ -0,0 +1,315 @@
+package edgex_foudry
```

Comment:
please add license header

Reply:
Thanks for reminding, the license header is added.